spk/sabnzbd/src/service-setup.sh

```diff
@@ -33,9 +35,9 @@ service_postinst ()
 
         # Create logs directory, otherwise it does not start due to permissions errors
         mkdir -p "$(dirname ${LOG_FILE})"
-        mkdir -p "${shared_folder}/incomplete"
-        mkdir -p "${shared_folder}/complete"
-        mkdir -p "${shared_folder}/watch"
+        install -m 0775 -o ${EFF_USER} -g ${GROUP} -d "${shared_folder}/incomplete"
```

Comment:
@hgy59 Does this not break ACL permissions by forcing Linux-style permissions?

Reply:
The end result is identical to the mkdir being theses sub-directories are in `Linux mode` with no ACL under DSM7.  As such it needs the `0775` + `${GROUP}` to be set properly otherwise it won't work for other packages.